si_ps_conv_fastcorrsApp/Db/kicks.template

```diff
@@ -0,0 +1,219 @@
+record(ao, "out-SI-$(SEC)$(DIP):PS-$(ORI):Current-SP") {
+    field(EGU, "A")
+    field(PREC, 3)
+    field(OUT, "SI-$(SEC)$(DIP):PS-$(ORI):Current-SP CA")
+}
+
+record(ao, "$(TEST)SI-$(SEC)$(DIP):PS-$(ORI):Kick-SP") {
+    field(EGU, "urad")
+    field(PREC, 3)
+    field(DTYP, "Python Device")
+    field(OUT, "@strn2curr $(SEC) $(DIP) $(ORI) Current-SP")
+}
+
+record(ai, "copy-SI-$(SEC)$(DIP):PS-$(ORI):Current-SP") {
+    field(EGU, "A")
+    field(PREC, 3)
+    field(SCAN, "Passive")
+    field(FLNK, "$(TEST)SI-$(SEC)$(DIP):PS-$(ORI):KickSP-RB PP")
+}
+
+record(ai, "$(TEST)SI-$(SEC)$(DIP):PS-$(ORI):KickSP-RB") {
+    field(EGU, "urad")
+    field(PREC, 3)
+    field(DTYP, "Python Device")
+    field(INP, "@curr2strn $(SEC) $(DIP) $(ORI) Current-SP")
+    field(SCAN, "Passive")
+    field(FLNK, "$(TEST)fwd-SI-$(SEC)$(DIP):PS-$(ORI):KickSP-RB")
+}
+
+record(ao, "$(TEST)fwd-SI-$(SEC)$(DIP):PS-$(ORI):KickSP-RB") {
+    field(EGU, "urad")
+    field(PREC, 3)
+    field(DOL, "$(TEST)SI-$(SEC)$(DIP):PS-$(ORI):KickSP-RB")
+    field(OMSL, "closed_loop")
+    field(OUT, "$(TEST)SI-$(SEC)$(DIP):PS-$(ORI):Kick-SP PP")
+}
```

Comment:
It would be nice to have a comment like: "this is done to update `Kick-SP` when `Current-SP` is updated by someone else"